app/src/main/java/com/kabouzeid/gramophone/ui/fragments/player/card/CardPlayerFragment.java

```diff
@@ -333,6 +324,7 @@ protected void onPostExecute(String lyrics) {
                     }
                 } else {
                     lyricsInfo = new LyricsDialog.LyricInfo(song.title, lyrics);
+                    playerAlbumCoverFragment.setSynchronizedLyrics(AbsSynchronizedLyrics.parse(lyrics));
```

Comment:
I don't think it's a good idea because `LyricInfo` is inner class of `LyricsDialog` and it is not used anywhere else. Maybe `LyricInfo` should be extracted as separate class and should be superclass of both static and synchronized lyrics?